PP queue breaks if push of a bundle fails

### Describe the bug
PP queue doesn't show all bundles that have been pushed.

### To Reproduce
Steps to reproduce the behavior:
1. Run the following queries against the DB:
```
insert into publishing_queue_audit(bundle_id, status, status_pojo, status_updated, create_date) VALUES ('01GDP522N7BZA7EP64HNRHQ7MN',9,'<com.dotcms.publisher.business.PublishAuditHistory>        +
   <endpointsMap>                                           +
     <entry>                                                +
       <string>192.168.131.94</string>                      +
       <map>                                                +
         <entry>                                            +
           <string>192.168.131.94</string>                  +
           <com.dotcms.publisher.business.EndpointDetail>   +
             <status>9</status>                             +
             <info>Everything ok</info>                     +
           </com.dotcms.publisher.business.EndpointDetail>  +
         </entry>                                           +
       </map>                                               +
     </entry>                                               +
   </endpointsMap>                                          +
   <publishStart>2022-09-23 21:46:46.467 UTC</publishStart> +
   <publishEnd>2022-09-23 21:46:48.220 UTC</publishEnd>     +
   <numTries>0</numTries>                                   +
   <assets>                                                 +
     <entry>                                                +
       <string>113ac47f-f044-44fb-827a-9db32472ff66</string>+
       <string>contentlet</string>                          +
     </entry>                                               +
   </assets>                                                +
 </com.dotcms.publisher.business.PublishAuditHistory>
','2022-09-23 21:46:36.41+00','2022-09-23 21:46:36.41+00');

-- This is the one breaking it
insert into publishing_queue_audit(bundle_id, status, status_pojo, status_updated, create_date) VALUES ('01GDP42MDCBTAMWG56C6Y35HKS',8,'<com.dotcms.publisher.business.PublishAuditHistory>                                                                                                                                                                                                                                                                                                    +
   <endpointsMap>                                                                                                                                                                                                                                                                                                                                       +
     <entry>                                                                                                                                                                                                                                                                                                                                            +
       <string>192.168.131.94</string>                                                                                                                                                                                                                                                                                                                  +
       <map>                                                                                                                                                                                                                                                                                                                                            +
         <entry>                                                                                                                                                                                                                                                                                                                                        +
           <string>192.168.131.94</string>                                                                                                                                                                                                                                                                                                              +
           <com.dotcms.publisher.business.EndpointDetail>                                                                                                                                                                                                                                                                                               +
             <status>8</status>                                                                                                                                                                                                                                                                                                                         +
             <info>Failed to publish because an error occurred: An error occurred when processing Contentlet in &apos;/data/shared/assets/bundles/01GDP42MDCBTAMWG56C6Y35HKS/live/www.lennox.com/1/1103-content.48a22979-4591-403c-b922-f4871057aee0.content.xml&apos; with ID &apos;7b1a5be8-8dea-4af1-8693-ad907fb29ca9&apos;: &apos;null&apos;</info>+
           </com.dotcms.publisher.business.EndpointDetail>                                                                                                                                                                                                                                                                                              +
         </entry>                                                                                                                                                                                                                                                                                                                                       +
       </map>                                                                                                                                                                                                                                                                                                                                           +
     </entry>                                                                                                                                                                                                                                                                                                                                           +
   </endpointsMap>                                                                                                                                                                                                                                                                                                                                      +
   <publishStart>2022-09-23 21:39:01.251 UTC</publishStart>                                                                                                                                                                                                                                                                                             +
   <publishEnd>2022-09-23 21:46:46.193 UTC</publishEnd>                                                                                                                                                                                                                                                                                                 +
   <numTries>0</numTries>                                                                                                                                                                                                                                                                                                                               +
 </com.dotcms.publisher.business.PublishAuditHistory>
','2022-09-23 21:39:00.382+00','2022-09-23 21:39:00.382+00');
```

2. Navigate to the PP portlet > Status/History
3. Should show 2 bundles
4. This error shows up in the log:
```
26/09/22 18:01:23:764 UTC] ERROR servlet.VisitorFilter: An exception occurred processing [/html/portlet/ext/contentlet/publishing/view_publish_audit_list.jsp] at line [314]

311: Map<String, String> bundleAssets = c.getStatusPojo() == null ? new HashMap<String, String>() : c.getStatusPojo().getAssets();
312:
313: PermissionableProxy pp = new PermissionableProxy();
314: for(String key : bundleAssets.keySet()) {
315:
316: String identifier = key;
317: String assetType = bundleAssets.get(key);


Stacktrace:
org.apache.jasper.JasperException: An exception occurred processing [/html/portlet/ext/contentlet/publishing/view_publish_audit_list.jsp] at line [314]

311: Map<String, String> bundleAssets = c.getStatusPojo() == null ? new HashMap<String, String>() : c.getStatusPojo().getAssets();
312:
313: PermissionableProxy pp = new PermissionableProxy();
314: for(String key : bundleAssets.keySet()) {
315:
316: String identifier = key;
317: String assetType = bundleAssets.get(key);


Stacktrace:
at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:626) ~[jasper.jar:9.0.41]
at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:515) ~[jasper.jar:9.0.41]
at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
Caused by: java.lang.NullPointerException
at org.apache.jsp.html.portlet.ext.contentlet.publishing.view_005fpublish_005faudit_005flist_jsp._jspService(view_005fpublish_005faudit_005flist_jsp.java:951) ~[?:?]
at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
... 74 more
```


### Expected behavior
The PP queue should list all the bundles regardless they fail or not.

Customer ticket: https://dotcms.zendesk.com/agent/tickets/108561

Reproducible in 22.03.2 LTS

--- a/dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_audit_detail.jsp
+++ b/dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_audit_detail.jsp
@@ -147,7 +147,7 @@
     </tr>
     <tr>
         <th><b><%= LanguageUtil.get(pageContext, "publisher_Audit_Asset_Number") %>: </b></th>
-        <td style="background: white"><%=currentEndpointHistory.getAssets().size() %></td>
+        <td style="background: white"><%=currentEndpointHistory.getAssets()==null ? 0 : currentEndpointHistory.getAssets().size() %></td>
 
     </tr>
 </table>

--- a/dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_audit_list.jsp
+++ b/dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_audit_list.jsp
@@ -308,7 +308,7 @@
 
 			//Check bundle permissions
 			PermissionAPI permAPI = APILocator.getPermissionAPI();
-			Map<String, String> bundleAssets = c.getStatusPojo() == null ? new HashMap<String, String>() : c.getStatusPojo().getAssets();
+			Map<String, String> bundleAssets = c.getStatusPojo() == null || c.getStatusPojo().getAssets() == null ? new HashMap<String, String>() : c.getStatusPojo().getAssets();
 
 			PermissionableProxy pp = new PermissionableProxy();
 			for(String key : bundleAssets.keySet()) {
@@ -414,7 +414,7 @@
                     new PublishQueueElementTransformer();
 
             for(PublishAuditStatus publishAuditStatus : iresults) {
-                final Map<String,String> assets = publishAuditStatus.getStatusPojo().getAssets();
+                final Map<String,String> assets = publishAuditStatus.getStatusPojo().getAssets() == null ? new HashMap<String, String>() : publishAuditStatus.getStatusPojo().getAssets();
 
                 final List<Map<String,Object>> assetsTransformed = assets.entrySet().stream()
                     .map(entry -> publishQueueElementTransformer.getMap(entry.getKey(), entry.getValue()))

